packages/grafana-runtime/src/services/EchoSrv.ts

```diff
@@ -24,17 +24,21 @@ export interface EchoMeta {
    */
   sessionId: string;
   /**
-   * The current users username used to login into Grafana e.g. email.
+   * The current user's username used to login into Grafana e.g. email.
    */
   userLogin: string;
   /**
-   * The current users unique identifier.
+   * The current user's unique identifier.
    */
   userId: number;
   /**
    * True when user is logged in into Grafana.
    */
   userSignedIn: boolean;
+  /**
+   * Current user's role
+   */
+  orgRole: string | '';
```

Comment:
in [ContextSrv](https://github.com/grafana/grafana/blob/main/public/app/core/services/context_srv.ts#L36) the type is `orgRole: OrgRole | '';`

Reply:
I think `OrgRole | ''` is more appropriate :)